src/vs/workbench/api/common/positron/extHost.positron.protocol.ts

```diff
@@ -53,6 +53,7 @@ export interface MainThreadLanguageRuntimeShape extends IDisposable {
 	$interruptSession(handle: number): Promise<void>;
 	$focusSession(handle: number): void;
 	$getSessionVariables(handle: number, accessKeys?: Array<Array<string>>): Promise<Array<Array<Variable>>>;
+	$getSessionVariableDataSummaries(handle: number, accessKeys: Array<Array<string>>): Promise<Array<string>>;
```

Comment:
I was thinking that to avoid a proliferation of new APIs related to data analysis tool calling, that we could potentially add a "thicker" API here with multiple options, something like:

`querySessionVariable(handle, accessKey, queryType, queryParams)`

where for now `queryType` could be one of `"schema"`, `"summary_stats"`, etc. Then we can add other types of tools without having to touch the API plumbing each time. There can still be multiple tools but the tool calls can all get "routed" through this "thick" API. Let me know what you think